adafruit_oauth2.py

```diff
@@ -52,13 +57,13 @@ class OAuth2:  # pylint: disable=too-many-arguments, too-many-instance-attribute
 
     def __init__(
         self,
-        requests,
-        client_id,
-        client_secret,
-        scopes,
-        access_token=None,
-        refresh_token=None,
-    ):
+        requests: adafruit_requests.Session,
```

Comment:
This is awesome!  Want to add this type annotation to the docstring?  You can do so by adding the type after the `param` keyword:

```
:param adafruit_requests.Session requests: An adafruit_requests object.
```

Reply:
done with latest commit